### PR TITLE
Added CreationDate and Size properties to Item

### PR DESCRIPTION
**Pull request by Gruppio**
Hi! 
I've added these 2 new properties to Item.
These new properties are optional because on some Linux systems may not be available

### CODE REVIEW

File: Sources/Files.swift [hunk 1]
```diff
@@ -197,7 +197,13 @@ public class FileSystem {
 
         /// The date when the item was last modified
         public private(set) lazy var modificationDate: Date = self.loadModificationDate()
-
+        
+        /// The date when the item was created, or `nil` if the information is not available
+        public private(set) lazy var creationDate: Date? = self.loadCreationDate()
+        
+        /// The size of the item in bytes, or `nil` if the information is not available
+        public private(set) lazy var size: UInt64? = self.loadSize()
```
Comment (clayellis) [inline Sources/Files.swift]: It would be preferred to expose `size` as an `Int` (as opposed to `UInt64`).

**Review by clayellis — CHANGES_REQUESTED** [review]
@Gruppio these are great additions. I left one comment on `size`.

File: Sources/Files.swift (hunk 1 above)
Comment (clayellis) [reply Sources/Files.swift]: Also, of note, @JohnSundell's [most recent article](https://www.swiftbysundell.com/posts/writing-self-documenting-swift-code) suggests that `size` could be more "self-documenting" by creating a `typealias` for `Int` called `ByteCount` to explicitly state that `size` is in terms of bytes.

Comment (Gruppio) [reply Sources/Files.swift]: Thanks @clayellis Great advice!

**Comment by JohnSundell** [comment]
Thanks for this @Gruppio! 😄 I'm wondering if it's worth force-unwrapping those optionals (like we do for `creationDate`) to make the API more simple. You mention that it may not be available on some Linux systems, will it always return `nil` on Linux, or just for some versions of it?

**Comment by Gruppio** [comment]
@JohnSundell I've made the changes but I had to disable those 2 test for linux 😞

**Pull request closed without merging**